src/main/java/com/gluonhq/utils/MavenArtifactResolver.java

```diff
@@ -77,9 +81,35 @@ public MavenArtifactResolver(List<Repository> repositories) {
         this.repositorySystem = createRepositorySystem();
         this.remoteRepositories = new LinkedList<>();
         repositories.forEach(r -> {
+            org.apache.maven.model.RepositoryPolicy releases = r.getReleases();
+            RepositoryPolicy releasesRepositoryPolicy = null;
+            if (releases != null) {
+                releasesRepositoryPolicy = new RepositoryPolicy(releases.isEnabled(),
+                        releases.getUpdatePolicy() == null ? UPDATE_POLICY_NEVER : releases.getUpdatePolicy(),
```

Comment:
Using `never` for the update policy means that it will only be checked once? I believe the default policy is `daily`?

Reply:
I added this check because when the repository has a null/empty policy, `setReleasePolicy(repositoryPolicy)` throws a warning and applies `never`. 
In any case, we don't care about updates, do we? We only apply this to find out an existing artifact, we are not going to update it later on.

Reply:
OK, for a release repository, it makes sense to use `never`. But I believe that for a snapshot repository, it will never check for a newer version after the first resolving has been done?

Reply:
We don't replace the regular use of maven repositories and artifact resolution that the pom.xml and settings.xml set.

We use `MavenArtifactResolver` just to get paths to given artifacts and build the classpath with them, in order to call Substrate: The live of this object is only through the use of the Client goals in the maven command line, therefore, the update policy doesn't really make sense in this case, does it?

Reply:
I believe it does make sense. Maven will store some information in the local repository so that it knows what update policy to apply, the next time the artifact needs to be resolved.